README.md

```diff
@@ -2,6 +2,9 @@
 
```

Comment:
- [ ] It would be better if our README is descriptive, for example, the project title should be `Create a database based on a diagram` instead of what we have currently